src/web/tools/PointerEventManager.ts

```diff
@@ -1,13 +1,13 @@
-import {
-  AdaptedEvent,
-  EventTypes,
-  MouseButton,
-  PointerType,
-} from '../interfaces';
+import { AdaptedEvent, EventTypes, MouseButton } from '../interfaces';
 import EventManager from './EventManager';
-import { isPointerInBounds } from '../utils';
+import { PointerTypeMapping, isPointerInBounds } from '../utils';
+import { PointerType } from '../../PointerType';
 
 const POINTER_CAPTURE_EXCLUDE_LIST = new Set<string>(['SELECT', 'INPUT']);
+const PointerTypes = {
```

Comment:
Basically same thing here, sometimes we use names like `touch/pen` while in other cases we use names like `finger/stylus`.

Reply:
Here `pen` is used because this is what we get inside `PointerEvents`. I could change it to `PointerTypes.Stylus` to match our convention. What do you think?

Reply:
As we discussed earlier, I've changed that to `Stylus` in order to stay consistent with our inner convention. Done in [6a92b89](https://github.com/software-mansion/react-native-gesture-handler/pull/2760/commits/6a92b89b170d91af9e09f5ae4f7d5abbb5c434d9).